test/api_test.js

```
@@ -130,4 +143,25 @@ suite('api', () => {
     result = await helper.github.isInstalledFor('unknownOwner', 'unknownRepo');
     assert.deepEqual(result, {installed: false});
   });
+
+  test('build badges', async function() {
+    await request.get('http://localhost:60415/v1/badge/abc123/coolRepo/master').end((err, res) => {
+      err ? console.log(err) : assert.equal(res.headers['content-length'], 928);
```

If there's an error, it will get sent to the console -- but the test will still succeed.  It's probably better to throw the error if it occurs.

Also, this is an odd combination of callback and `await`.  @imbstack I think you know more about superagent-promise than I do? Is this right?

---

Yeah, it's a little bit weird. We don't want to `console.log`. I'll comment here again soon when I figure out how to do this.

---

What if I put another assert.equal in there? For status code, for example. Then the test will fail in case of an error